FiveCalls/FiveCalls/IssuesViewController.swift

```diff
@@ -152,70 +175,93 @@ class IssuesViewController : UITableViewController {
     }
 
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if let typedInfo = R.segue.issuesViewController.moreSegue(segue: segue) {
+            typedInfo.destination.issuesManager = issuesManager
+        }
         guard let indexPath = tableView.indexPathForSelectedRow else { return }
         if let typedInfo = R.segue.issuesViewController.issueSegue(segue: segue) {
             typedInfo.destination.issuesManager = issuesManager
-            typedInfo.destination.issue = viewModel.issues[indexPath.row]
+            typedInfo.destination.issue = viewModel.categories[indexPath.section].issues[indexPath.row]
         }
     }
     
     // MARK: - UITableViewDataSource
     
     override func numberOfSections(in tableView: UITableView) -> Int {
-        return 1
+        // Empty tableview if we haven't fetched data yet.
+        if viewModel.categories.count == 0 {
```

Comment:
this is circumstantial code. I'd prefer do have an `isLoading` property or something similar that isn't tied to an arbitrary dictionary or array having data.

Reply:
Got it. Will change this.